packages/block-library/src/tag-cloud/edit.js

```diff
@@ -82,6 +96,54 @@ function TagCloudEdit( { attributes, setAttributes, taxonomies } ) {
 					max={ MAX_TAGS }
 					required
 				/>
+				<Flex>
+					<FlexItem isBlock>
+						<UnitControl
+							label={ __( 'Smallest size' ) }
+							value={ smallestFontSize }
+							onChange={ ( value ) => {
+								const unit = value.slice( -2 );
```

Comment:
Same goes here about the parsing with `-2`. We need to handle it more gracefully and dynamically.

Reply:
Agree 💯  ! Addressed this comment.